demo/tests/duplicate_seat.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Dużo tego typu testów. Może one nie były potrzebne do demo, a lepiej było umieścić je w testach automatycznych? Fajnie jakby to się automatycznie sprawdzało, a tutaj wystarczyłoby mniej.